README.md

```diff
@@ -60,11 +60,14 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Vegan Foods] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**[Vegan Foods]** is an onlne platform where users
+can query about their favorite vegan meal and get tips on how to cook it
+
+[Project tour](https://www.loom.com/share/d0c646e851ea4bc499394494818a25ae)
```

Comment:
As mentioned in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/5801583dd2ad132a9326d4bd429291d954e4e824/capstone/react_capstone.md?plain=1#L89) for the project, kindly record a video of maximum length, 5 minutes, where you present the features of the project and a highlight about a piece of the code the you find interesting or are proud of making.